common/src/models/domain/account.ts

```diff
@@ -0,0 +1,67 @@
+import { OrganizationData } from '../data/organizationData';
+
+import { KdfType } from '../../enums/kdfType';
+import { StorageKey } from '../../enums/storageKey';
+
+export class Account {
+    userId: string;
+    email: string;
+    settings: Map<string, any> = new Map<string, any>();
```

Comment:
- Would be nice to know the status of the account. (Active, locked, unlocked).
- Which server it's using.

Reply:
> Would be nice to know the status of the account. (Active, locked, unlocked).

Some of this was considered, but ultimately scraped. It seemed outside of the scope of an account to know about the existence of other accounts. There is also, at this point at least, only a possibility for the active account to be unlocked. All others will be locked. More details [here](https://app.asana.com/0/0/1200520645190695/1200873382578787/f).

> Which server it's using.

This is stored in settings and accessible that way. I'm reluctant to add class level properties here that aren't user identifiers, anything else should really be in the settings map.

Reply:
I added the server helper, status forecoming